instat/static/InstatObject/R/data_object_R6.R

```diff
@@ -2178,6 +2178,11 @@ DataSheet$set("public","split_date", function(col_name = "", year_val = FALSE, y
     col_name <- next_default_item(prefix = "day_in_month", existing_names = self$get_column_names(), include_index = FALSE)
     self$add_columns_to_data(col_name = col_name, col_data = day_in_month_vector)
   }
+  if(days_in_month) {
+    days_in_month_vector <- as.integer(lubridate::days_in_month(col_data))

```

Comment:
```suggestion
    days_in_month_vector <- as.numeric(lubridate::days_in_month(col_data))
```
`as.numeric` would be better, since we sometimes have problems with integers.

Reply:
The `day_in_month` above could also change to `as.numeric()`

Reply:
@Wycklife  just a polite reminder. Kindly make the necessary changes here. I see this will have conflict with the Climsoft but we will fix that later.

Reply:
Is there going to be a conflict? It shouldn't if the edits are on different parts of the file - its easy to resolve in R files anyway as you say.